todo-mvc/src/widgets/createTodoItem.ts

```diff
@@ -116,8 +116,8 @@ const createTodoItem: TodoItemFactory = createWidget
 					instance.on('statechange', manageChildren);
 
 					/* We have missed the widgets initial state change because we created the widgets async
-					 * so we should invlidate the instance, so it will re-calculate its children */
-					instance.invalidate();
+					 * so we should set state on the instance, so it will re-calculate its children */
+					instance.setState({});
```

Comment:
@agubler I tired `instance.invalidate()` but right now it causes a regression and the widget renders as "blank".  It must have to do something with the lifecycle of the intialization.  I think when we do some of the other PR we have planned for `dojo-widget` this will *fix* this issue, but for now we have to do something like this.